packages/botonic-react/tests/webchat/storage.test.jsx

```diff
@@ -13,9 +13,9 @@ describe('TEST: storage', () => {
       TestRenderer.create(<Webchat />)
     })
     const botonicState = JSON.parse(localStorage.getItem('botonicState'))
-    expect(botonicState.user).toHaveProperty('id' && 'name')
+    expect(botonicState.session.user).toHaveProperty('id' && 'name')
```

Comment:
Oh, cool! Jasmine does not have this `toHaveProperty` check! Still, is this expression correct to check for these two properties? Wouldn't it be... ?
```
expect(botonicState.session.user).toHaveProperty(['id', 'name'])
```

Reply:
it seems to not work well by passing it like an array. I've just leave it as is 👍